prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -736,7 +737,7 @@ class Hl7Serializer(
 
             // Override with organization name if available
             Hl7Configuration.OrderingFacilityName.ORGANIZATION_NAME -> {
-                val organizationName = report.getString(row, "organization_name") ?: rawFacilityName
+                val organizationName = report.getString(row, "organization_name").trimToNull() ?: rawFacilityName
```

Comment:
I have not seen this custom StringUtilities, but org.apache.commons.lang.StringUtils has a trimToNull that I think we should use instead.

Reply:
@carlosfelix2 I'm generally in favor of using libraries for things like this, but it feels like overkill to import the entire Apache library for a single function, especially when says library also includes a bunch of competing functions that are built into Kotlin. If we're going to use more than just `trimToNull` from StringUtils then we can import and migrate.